lib/kernel/doc/src/pg.xml

```diff
@@ -130,6 +130,18 @@
       </desc>
     </func>
 
+    <func>
+      <name name="start" arity="2" since="OTP 26.0"/>
+      <name name="start_link" arity="2" since="OTP 26.0"/>
+      <fsummary>Start additional scope with options.</fsummary>
+      <desc>
+        <p>Starts additional scope.</p>
+        <p><c>Options</c> are a map with optional keys. Currently
+          only the <c>node_type</c> key is supported, and it must be
+          one of <c>visible</c> (default), <c>hidden</c>, or <c>all</c>.</p>
```

Comment:
I think it's also necessary to add a `<datatype>` documentation that explains the difference between all/hidden/visible. Otherwise there is no way to know what this parameter means.

Reply:
Pushed better docs using taglist as in net_kernel:connect_node.